books/management/commands/addepub.py

```diff
@@ -249,7 +249,7 @@ def process_epub(self, filename, use_symlink=False):
                                 .split(',')
                             for tag in subject_split:
                                 if tag is not ' ':
-                                    tag = tag.encode("utf-8").lower().strip()
+                                    tag = tag.lower().strip()
```

Comment:
The change was indeed because there were a fair amount of opfs that complained they were in ascii I believe. The errors didn't rid themselves entirely, and I was still looking at how to deal with the various cases, though less errors showed up that way. How to handle various encodings is a good question.

Out of my two groups of ePubs, one I'd like keep as with my current deployment, but another also large but smaller set, I can run through again and see what errors show up.